tests/jasmine/test.js

```diff
@@ -1,4 +1,16 @@
+beforeAll(() => {})
+afterAll(() => {
+    throw new Error('afterAll failure in root scope')
```

Comment:
two questions:

1. shouldn't this fail the smoke tests?
2. if not can we somehow assert the spec report that it outputs proper logs?

Reply:
1. beforeAll / afterAll hook failures are not failing the suite
2.1. root scope hooks and tests don't appear in any reporters including spec (for jasmine)
2.2. don't know if we can validate spec reporter result, let me check that

Reply:
point 2.1 is a defect I think (not related to the PR)

Reply:
we can validate reporter output with `wdio-smoke-test-reporter`, let me do that

Reply:
@christian-bromann added smoke test with reporter and raised a defect https://github.com/webdriverio/webdriverio/issues/4582